### PR TITLE
#162948882 Users should be able to share articles on social media and via email

### PR DESCRIPTION
**Pull request by boltonotieno**
#### What does this PR do?
*  Implements sharing of articles on social media and via email

#### Description of Task to be completed?

- create share article UI with facebook, twitter, and email
- share articles through the UI

#### How should this be manually tested?
* Clone ah-alpha repo `git clone https://github.com/andela/ah-alpha.git`
* Checkout to the branch `ft-share-articles-162948882`
* Install the required dependencies using `npm install`
* Run the local development server using `npm start`
* Click on one article to view the social icons
* Click one of the social icons to share the viewed article

#### Screenshot
image

#### What are the relevant pivotal tracker stories?
[#162948882](https://www.pivotaltracker.com/story/show/162948882

### CODE REVIEW

File: src/components/containers/sharing/ShareIcons.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    super(props);
+    this.state = {
+    };
+  }
+
+  render() {
+    const template = "Highlights from Author's Haven";
+    const { shareURL, title, author } = this.props;
+    const shareTitle = `Take a look at this:  ${title}`;
+    const emailBody = `Hello\nTake a look at this article by ${author.charAt(0).toUpperCase() + author.substr(1).toLowerCase()} \n ${shareURL}`;
```
Comment (danuluma) [inline src/components/containers/sharing/ShareIcons.js]: This line is too long, kindly check and split

Comment (boltonotieno) [reply src/components/containers/sharing/ShareIcons.js]: Thanks for the feedback, I have refactored

**Review by Ruiru11 — APPROVED** [review]
The code looks good. The app is running good work

**Review by danuluma — APPROVED** [review]
Great work

**Review by Deekerubo — APPROVED** [review]

**Pull request merged**